app/controllers/clearance/passwords_controller.rb

```diff
@@ -72,7 +72,8 @@ def password_reset_params
       ActiveSupport::Deprecation.warn %{Since locales functionality was added, accessing params[:user] is no longer supported.}
       params[:user][:password]
     else
-      params[:password_reset][:password]
+      reset_params = params[:password_reset] || {}
+      reset_params[:password]
```

Comment:
Yeah, we support back to 1.9.3, so that's not an option here. I don't want to take a dependency for it.